R/descriptives.R

```diff
@@ -1541,3 +1555,92 @@ Descriptives <- function(jaspResults, dataset, options) {
 
   return(tb)
 }
+
+
+.descriptivesHeatmaps <- function(container, dataset, variables, options) {
+  axesNames <- c(options[["heatmapHorizontal"]], options[["heatmapVertical"]])
+
+  # we are not ready to plot
+  if(length(variables) == "" || any(axesNames == "")) return()
+
+  axes <- .readDataSetToEnd(columns.as.factor = encodeColNames(axesNames))
```

Comment:
JASP was protesting when I did not do `encodeColNames()`, but I assume that the variable names should already come from qml encoded, or not? The values come from a `DropDown{}`.

Reply:
@JorisGoosen the column names should be encoded, even if they come from a `DropDown`, right? But if @Kucharssim had to resort to this, I'm not sure if they are.

Reply:
They *should be* but it is possible there wasn't a proper meta added yet.
I'll make an issue for it.